packages/react-test-renderer/src/ReactTestRenderer.js

```diff
@@ -444,6 +446,9 @@ function create(element: React$Element<any>, options: TestRendererOptions) {
     if (options.unstable_isConcurrent === true) {
       isConcurrent = true;
     }
+    if (options.unstable_strictMode === true) {
+      strictModeLevel = 2;
+    }
```

Comment:
Oh interesting. Why the level conversion? Why not just pass through as a boolean that sets both `StrictLegacyMode` and `StrictEffectsMode` like `<StrictMode>` itself does now?

No big deal I guess. Just a little unexpected.

Reply:
I was trying to keep as much of the levels infra a possible in case we decide to use it. I didn't realize that the StrictMode component changed this too, so I'll update this!

Reply:
Updated!

Reply:
Cool 👍🏼